System.IO.Abstractions.TestingHelpers.Tests/MockFileInfoTests.cs

```diff
@@ -491,6 +491,24 @@ public void MockFileInfo_CopyTo_ThrowsExceptionIfSourceDoesntExist()
             Assert.Throws<FileNotFoundException>(action);
         }
 
+        [TestCase(@"..\..\..\c.txt")]
+        [TestCase(@"c:\a\b\c.txt")]
+        [TestCase(@"c:\a\c.txt")]
+        [TestCase(@"c:\c.txt")]
+        public void MockFileInfo_ToString_ShouldReturnOriginalFilePath(string path)
+        {
+            //Arrange
+            var filePath = XFS.Path(path);
+
+            //Act
+            var mockFileInfo = new MockFileInfo(new MockFileSystem(), filePath);
+            var realFileInfo = new FileInfo(filePath);
```

Comment:
Same comment as above, please remove this

Reply:
Removed real implementation in test